src/quick-lint-js/diag/diagnostic-types-2.h

```diff
@@ -3589,6 +3589,16 @@ struct Diag_TypeScript_Namespace_Alias_Cannot_Use_Import_Type {
                   ARG(type_keyword))]]  //
   Source_Code_Span type_keyword;
 };
+
+struct Diag_Ambiguous_Let_Call {
+  [[qljs::diag("E0720", Diagnostic_Severity::warning)]]  //
+  [
+      [qljs::message("ambiguous use of the keyword 'let'; "
+                     "let is a function in this scope",
```

Comment:
"ambiguous" is incorrect. All JS engines will consider `let` to be a function reference in this case; there is no ambiguity. Perhaps a better word is "confusing": "confusing use of the keyword 'let'; let is a function in this scope"

There's another problem with this message: It doesn't help the user solve the issue. So I propose wording such as the following:
> 'let' is treated as a function; remove parentheses to instead declare a variable